aws/scripts/cloud-init-cratedb-rpm.tftpl

```diff
@@ -11,9 +11,15 @@ yum_repos:
     gpgcheck: false
     gpgkey: https://packagecloud.io/prometheus-rpm/release/gpgkey
     name: prometheus-rpm
+  # workaround for crate-release RPM package not being compatible with AL2023 as of now
+  cratedb:
+    baseurl: https://cdn.crate.io/downloads/yum/7/$basearch
+    enabled: true
+    gpgcheck: true
+    gpgkey: https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
+    name: CrateDB stable releases
 
 packages:
-  - curl
```

Comment:
Amazon Linux 2023 comes with `curl-minimal` already installed, trying to install `curl` as well results in a package conflict (both providing the same binary).